formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/FormstackService.scala

```diff
@@ -79,24 +79,33 @@ object FormstackService extends FormstackRequestService with LazyLogging {
     submissionIdEmails: List[SubmissionIdEmail],
     accountToken: FormstackAccountToken,
     encryptionPassword: String): Either[Throwable, List[Submission]] = {
-    submissionIdEmails.traverse { submissionIdEmail =>
+    val submissionResults: Either[Throwable, List[Option[Submission]]] = submissionIdEmails.traverse { submissionIdEmail =>
       val response =
         Http(s"https://www.formstack.com/api/v2/submission/${submissionIdEmail.submissionId}.json")
           .header("Authorization", accountToken.secret)
           .param("encryption_password", encryptionPassword)
           .asString
 
-      if(!response.is2xx) {
-        logger.error(response.body)
-      }
+      def isErrorResponse = !response.is2xx
+
+      // Define errors that we can safely skip
+      def isDecryptionError = response.body.contains("An error occurred while decrypting the submissions")
+      def isSubmissionDeletedError = response.body.contains("An error occurred while decrypting the submissions")
+      def isFormNotFoundError = response.body.contains("A valid form could not be found")
 
-      /* There are a couple of forms that the Formstack API can't seem to decrypt. There seems to be no way around this
-      *  so we capture this specific error and skip these forms. */
-      if(response.body.contains("An error occurred while decrypting the submission"))
-        Left(FormstackDecryptionError(s"${response.body} | submission id: ${submissionIdEmail.submissionId}"))
-      else
-        decode[Submission](response.body)
+      def isSkippableError = isErrorResponse && isDecryptionError && isSubmissionDeletedError && isFormNotFoundError
```

Comment:
It should be one of the skippable errors, not all at the same time
```suggestion
      def isSkippableError = isErrorResponse && (isDecryptionError || isSubmissionDeletedError || isFormNotFoundError)
```